lib/index.js

```diff
@@ -5,5 +5,9 @@ var parse = require('./parse');
 
 module.exports = {
     parse: parse,
+    // String constants
```

Comment:
let's group these under a `formats` object, so it's clear that "parse" isn't a format, and so the full list can be obtained via reflection.